content/image.js

```diff
@@ -45,6 +45,24 @@ function findByURL(url) {
   return find(urlToFilePath(url));
 }
 
+function findByURLWithFallback(url) {
+  let filePath = findByURL(url);
+  const locale = url.split("/")[1].toLowerCase();
```

Comment:
If you assign the urlParts, you can use them instead of the `url.replace` below:

```suggestion
  const urlParts = url.split("/")
  const locale = urlParts[1].toLowerCase();
```